lib/filesystem.nix

```diff
@@ -54,4 +54,24 @@
       dir + "/${name}"
   ) (builtins.readDir dir));
 
+  # find: String -> Path -> [ Path ]
+  #
+  # Given a suffix and directory, return a filtered list containing the full
+  # path to all files matching the given suffix recursively.
+  #
+  # Example:
+  #   find ".csv" "${glibc.dev}/include/protocols"
```

Comment:
```suggestion
  #   find ".h" "${glibc.dev}/include/protocols"
```

Reply:
Ah derp. I had been using this for CSVs and copy pasted my example, lol. Thank you!